### PR TITLE
 Support Ubuntu 24.04

### PR DESCRIPTION
**Pull request by san983**
Closes https://github.com/fullstaq-ruby/server-edition/issues/139

Followed https://github.com/fullstaq-ruby/server-edition/blob/main/dev-handbook/add-new-distro.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -457,7 +457,7 @@ sudo yum install fullstaq-ruby-3.1.1
 ### Debian/Ubuntu
 
  * Supported Debian versions: 11 *(bullseye)*, 10 *(buster)*, 9 *(stretch)*
- * Supported Ubuntu versions: 22.04, 20.04, 18.04
```
Comment (san983) [inline README.md:460]: 18.04 is already EOL and no longer in this repo

File: config.yml [hunk 2]
```diff
@@ -119,6 +120,7 @@ distribution_exclusions:
     distros:
       - debian-12
       - ubuntu-22.04
+      - ubuntu-24.04
```
Comment (san983) [inline config.yml:123]: Assuming as if 3.0 is a no go in 22.04, it will also be for 24.04

**Comment by FooBarWidget** [comment]
Thanks, this looks good to me. Could you rebase your pull request?

**Comment by san983** [comment]
> Thanks, this looks good to me. Could you rebase your pull request?

Sure thing, done

**Comment by FooBarWidget** [comment]
Merged. Thanks @san983 !

**Comment by joesiewert** [comment]
I'm getting this when trying to grab the new version. Anyone else?

```

**Comment by joesiewert** [comment]
I'm getting this when trying to grab the new version. Anyone else?

```
 The repository 'https://apt.fullstaqruby.org ubuntu-24.04 Release' does not have a Release file.
```

**Comment by G-Rath** [comment]
@joesiewert we're hitting this too :/

**Comment by FooBarWidget** [comment]
Should be fixed now.

**Pull request merged**